docs/openapi.yaml

```diff
@@ -70,6 +70,199 @@ paths:
               schema:
                 $ref: "#/components/schemas/TicketSearchResponse"
 
+  /resources2:
+    get:
+      description: "Search for public available publications by given criteria"
+      tags:
+        - external
+      parameters:
+      - in: query
+        name: category
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: contributor
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: created_before
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: created_since
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: doi
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: fields
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: funding
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: funding_source
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: id
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: institution
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: issn
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: lang
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: modified_before
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: modified_since
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: order
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: page
+        description: "Term to search for in resource."
+        required: false
+        schema:
+          type: string
+      - in: query
+        name: per_page
```

Comment:
is search terms copy paste from cristin? if that is the case, should we use our own terms or keep cristin terms?

Reply:
I designed it to be equal, but it is easy to extend to validate new terms as well.
i.e. ` PER_PAGE(NUMBER, EQUALS, "per_page", "results", "per_page|results", null),`
will accept 'per_page' or 'results'